scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,24 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
+  while read -r line; do
+    comment=$(echo "$line" | cut -c1)
```

Comment:
yeah... my bad... :blush: 

this will (tested with busybox static):
```
grep -v ^# "${file_to_read}" | while read -r line; do
        <do stuff with line>
done
```